type-annotator-scanner/src/test/java/edu/ucr/cs/riple/scanner/tools/TrackerNodeDisplay.java

```diff
@@ -29,14 +29,14 @@
 public class TrackerNodeDisplay implements Display {
 
   public final String callerClass;
-  public final String callerMethod;
+  public final String callerMember;
   public final String calleeClass;
   public final String member;
```

Comment:
Yes, they it should be `calleeMember` and yes it can be a field as well. I understand that `calleeMember` is not accurate as it can be a `field` as well. I am going to change it for now to `calleeMember` but it should be changed in future. 851f8ebc67f28bf42122f9a51a4df1f57639545c

Reply:
I _think_ "caller" here refers to the region containing a use of some member, and "callee" refers to the used member.  So maybe `regionClass`, `regionMember`, `usedClass`, `usedMember`?  @nimakarimipour if you agree we can just make the change.  Otherwise, we can leave a TODO comment and fix in a follow-up.

Reply:
@msridhar Sure, we are already renaming a member of this class in this PR, I will update it now. Thank you for the suggestions.

Reply:
bc2ad4e1c4dfe927a6a172e03c82140f2908700b